
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Aui(object):
    def setupUi(self, Aui):
        Aui.setObjectName("Aui")
        Aui.resize(960, 750)
        Aui.setMinimumSize(QtCore.QSize(960, 640))
        Aui.setMaximumSize(QtCore.QSize(960, 640))
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(8)
        Aui.setFont(font)
        Aui.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.layoutWidget = QtWidgets.QWidget(Aui)
        self.layoutWidget.setGeometry(QtCore.QRect(30, 20, 900, 600))
        self.layoutWidget.setObjectName("layoutWidget")
        self.verticalLayout_7 = QtWidgets.QVBoxLayout(self.layoutWidget)
        self.verticalLayout_7.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout_7.setObjectName("verticalLayout_7")
        spacerItem = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_7.addItem(spacerItem)
        self.TitleLabel = TitleLabel(self.layoutWidget)
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(24)
        font.setBold(False)
        font.setWeight(50)
        self.TitleLabel.setFont(font)
        self.TitleLabel.setTabletTracking(False)
        self.TitleLabel.setFocusPolicy(QtCore.Qt.StrongFocus)
        self.TitleLabel.setObjectName("TitleLabel")
        self.verticalLayout_7.addWidget(self.TitleLabel, 0, QtCore.Qt.AlignHCenter|QtCore.Qt.AlignVCenter)
        spacerItem1 = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_7.addItem(spacerItem1)
        self.SimpleCardWidget = SimpleCardWidget(self.layoutWidget)
        self.SimpleCardWidget.setMinimumSize(QtCore.QSize(736, 160))
        self.SimpleCardWidget.setMaximumSize(QtCore.QSize(704, 160))
        self.SimpleCardWidget.setObjectName("SimpleCardWidget")
        self.BodyLabel_3 = BodyLabel(self.SimpleCardWidget)
        self.BodyLabel_3.setGeometry(QtCore.QRect(9, -1, 736, 161))
        self.BodyLabel_3.setMinimumSize(QtCore.QSize(736, 0))
        self.BodyLabel_3.setMaximumSize(QtCore.QSize(704, 16777215))
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(10)
        font.setBold(False)
        font.setUnderline(False)
        font.setWeight(50)
        self.BodyLabel_3.setFont(font)
        self.BodyLabel_3.setObjectName("BodyLabel_3")
        self.verticalLayout_7.addWidget(self.SimpleCardWidget, 0, QtCore.Qt.AlignHCenter|QtCore.Qt.AlignVCenter)
        spacerItem2 = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_7.addItem(spacerItem2)
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setObjectName("horizontalLayout")
        spacerItem3 = QtWidgets.QSpacerItem(64, 0, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem3)
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setObjectName("verticalLayout")
        self.CardWidget = CardWidget(self.layoutWidget)
        self.CardWidget.setMinimumSize(QtCore.QSize(192, 192))
        self.CardWidget.setMaximumSize(QtCore.QSize(192, 192))
        self.CardWidget.setObjectName("CardWidget")
        self.IconWidget_4 = IconWidget(self.CardWidget)
        self.IconWidget_4.setGeometry(QtCore.QRect(0, 0, 192, 192))
        self.IconWidget_4.setMinimumSize(QtCore.QSize(192, 192))
        self.IconWidget_4.setMaximumSize(QtCore.QSize(192, 192))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("./image/可莉.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.IconWidget_4.setIcon(icon)
        self.IconWidget_4.setObjectName("IconWidget_4")
        self.verticalLayout.addWidget(self.CardWidget)
        spacerItem4 = QtWidgets.QSpacerItem(0, 8, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout.addItem(spacerItem4)
        self.ElevatedCardWidget_1 = ElevatedCardWidget(self.layoutWidget)
        self.ElevatedCardWidget_1.setMinimumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_1.setMaximumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_1.setBorderRadius(5)
        self.ElevatedCardWidget_1.setObjectName("ElevatedCardWidget_1")
        self.verticalLayout_8 = QtWidgets.QVBoxLayout(self.ElevatedCardWidget_1)
        self.verticalLayout_8.setObjectName("verticalLayout_8")
        self.CaptionLabel_11 = CaptionLabel(self.ElevatedCardWidget_1)
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(10)
        font.setBold(False)
        font.setWeight(50)
        self.CaptionLabel_11.setFont(font)
        self.CaptionLabel_11.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.CaptionLabel_11.setMouseTracking(True)
        self.CaptionLabel_11.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.CaptionLabel_11.setObjectName("CaptionLabel_11")
        self.verticalLayout_8.addWidget(self.CaptionLabel_11, 0, QtCore.Qt.AlignHCenter|QtCore.Qt.AlignVCenter)
        self.verticalLayout.addWidget(self.ElevatedCardWidget_1)
        spacerItem5 = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout.addItem(spacerItem5)
        spacerItem6 = QtWidgets.QSpacerItem(5, 0, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        self.verticalLayout.addItem(spacerItem6)
        self.horizontalLayout.addLayout(self.verticalLayout)
        spacerItem7 = QtWidgets.QSpacerItem(64, 0, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem7)
        self.verticalLayout_2 = QtWidgets.QVBoxLayout()
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.CardWidget_2 = CardWidget(self.layoutWidget)
        self.CardWidget_2.setMinimumSize(QtCore.QSize(192, 192))
        self.CardWidget_2.setMaximumSize(QtCore.QSize(192, 192))
        self.CardWidget_2.setObjectName("CardWidget_2")
        self.IconWidget_5 = IconWidget(self.CardWidget_2)
        self.IconWidget_5.setGeometry(QtCore.QRect(0, 0, 192, 192))
        self.IconWidget_5.setMinimumSize(QtCore.QSize(192, 192))
        self.IconWidget_5.setMaximumSize(QtCore.QSize(192, 192))
        self.IconWidget_5.setIcon(icon)
        self.IconWidget_5.setObjectName("IconWidget_5")
        self.verticalLayout_2.addWidget(self.CardWidget_2)
        spacerItem8 = QtWidgets.QSpacerItem(0, 8, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_2.addItem(spacerItem8)
        self.ElevatedCardWidget_2 = ElevatedCardWidget(self.layoutWidget)
        self.ElevatedCardWidget_2.setMinimumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_2.setMaximumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_2.setBorderRadius(5)
        self.ElevatedCardWidget_2.setObjectName("ElevatedCardWidget_2")
        self.verticalLayout_9 = QtWidgets.QVBoxLayout(self.ElevatedCardWidget_2)
        self.verticalLayout_9.setObjectName("verticalLayout_9")
        self.CaptionLabel_12 = CaptionLabel(self.ElevatedCardWidget_2)
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(10)
        font.setBold(False)
        font.setWeight(50)
        self.CaptionLabel_12.setFont(font)
        self.CaptionLabel_12.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.CaptionLabel_12.setMouseTracking(True)
        self.CaptionLabel_12.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.CaptionLabel_12.setObjectName("CaptionLabel_12")
        self.verticalLayout_9.addWidget(self.CaptionLabel_12, 0, QtCore.Qt.AlignHCenter|QtCore.Qt.AlignVCenter)
        self.verticalLayout_2.addWidget(self.ElevatedCardWidget_2)
        spacerItem9 = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_2.addItem(spacerItem9)
        spacerItem10 = QtWidgets.QSpacerItem(5, 0, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        self.verticalLayout_2.addItem(spacerItem10)
        self.horizontalLayout.addLayout(self.verticalLayout_2)
        spacerItem11 = QtWidgets.QSpacerItem(64, 0, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem11)
        self.verticalLayout_3 = QtWidgets.QVBoxLayout()
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.CardWidget_3 = CardWidget(self.layoutWidget)
        self.CardWidget_3.setMinimumSize(QtCore.QSize(192, 192))
        self.CardWidget_3.setMaximumSize(QtCore.QSize(192, 192))
        self.CardWidget_3.setObjectName("CardWidget_3")
        self.IconWidget_6 = IconWidget(self.CardWidget_3)
        self.IconWidget_6.setGeometry(QtCore.QRect(0, 0, 192, 192))
        self.IconWidget_6.setMinimumSize(QtCore.QSize(192, 192))
        self.IconWidget_6.setMaximumSize(QtCore.QSize(192, 192))
        self.IconWidget_6.setIcon(icon)
        self.IconWidget_6.setObjectName("IconWidget_6")
        self.verticalLayout_3.addWidget(self.CardWidget_3)
        spacerItem12 = QtWidgets.QSpacerItem(0, 8, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_3.addItem(spacerItem12)
        self.ElevatedCardWidget_3 = ElevatedCardWidget(self.layoutWidget)
        self.ElevatedCardWidget_3.setMinimumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_3.setMaximumSize(QtCore.QSize(192, 32))
        self.ElevatedCardWidget_3.setBorderRadius(5)
        self.ElevatedCardWidget_3.setObjectName("ElevatedCardWidget_3")
        self.verticalLayout_10 = QtWidgets.QVBoxLayout(self.ElevatedCardWidget_3)
        self.verticalLayout_10.setObjectName("verticalLayout_10")
        self.CaptionLabel_13 = CaptionLabel(self.ElevatedCardWidget_3)
        font = QtGui.QFont()
        font.setFamily("萝莉体")
        font.setPointSize(10)
        font.setBold(False)
        font.setWeight(50)
        self.CaptionLabel_13.setFont(font)
        self.CaptionLabel_13.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.CaptionLabel_13.setMouseTracking(True)
        self.CaptionLabel_13.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.CaptionLabel_13.setObjectName("CaptionLabel_13")
        self.verticalLayout_10.addWidget(self.CaptionLabel_13, 0, QtCore.Qt.AlignHCenter|QtCore.Qt.AlignVCenter)
        self.verticalLayout_3.addWidget(self.ElevatedCardWidget_3)
        spacerItem13 = QtWidgets.QSpacerItem(0, 32, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.verticalLayout_3.addItem(spacerItem13)
        spacerItem14 = QtWidgets.QSpacerItem(5, 0, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        self.verticalLayout_3.addItem(spacerItem14)
        self.horizontalLayout.addLayout(self.verticalLayout_3)
        spacerItem15 = QtWidgets.QSpacerItem(64, 0, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem15)
        self.verticalLayout_7.addLayout(self.horizontalLayout)

        self.retranslateUi(Aui)
        QtCore.QMetaObject.connectSlotsByName(Aui)

    def retranslateUi(self, Aui):
        _translate = QtCore.QCoreApplication.translate
        Aui.setWindowTitle(_translate("Aui", "Form"))
        self.TitleLabel.setText(_translate("Aui", "选择地图画类型"))
        self.BodyLabel_3.setText(_translate("Aui", "             玩家们会用地图来创作像素画。默认的地图大小为 128×128 并会读取每一个方块为一种特定的颜色。通过以特\n"
"      定的排列放置方块，为创作像素画图像提供了可能。创作地图像素画有两种方法：平面型和阶梯型。平面型方法涉\n"
"      及在一个平面上放置像素图像，最后创建出一个地板。平面型方法是两种方法中较简单的一种，但只提供了60色的\n"
"      调色板。而阶梯型方法提供了180色，但使用起来更加复杂了。在阶梯型方法中，方块需要被放置不同的高度来获\n"
"      得特定的颜色变化。一个方块的颜色会在比北边方块低的位置时变暗，而在比北边方块高的位置时变亮。使用平面\n"
"      型方法时，可以使用以上列表中每个颜色组的第二个阴影。对于阶梯型方法，可以使用以上颜色组的前三个阴影。\n"
"      只有使用外部工具才能获得第四个阴影。"))
        self.CaptionLabel_11.setText(_translate("Aui", "平 面 地 图 画"))
        self.CaptionLabel_12.setText(_translate("Aui", "立 体 地 图 画"))
        self.CaptionLabel_13.setText(_translate("Aui", "纯 文 件 地 图 画"))
from qfluentwidgets import BodyLabel, CaptionLabel, CardWidget, ElevatedCardWidget, IconWidget, SimpleCardWidget, TitleLabel
